code/datums/keybinding/mob.dm

```diff
@@ -146,18 +146,64 @@
 		user.mob.dropItemToGround(I)
 	return TRUE
 
-/datum/keybinding/mob/toggle_move_intent
+
+/datum/keybinding/mob/examine
 	key = "Alt"
-	name = "toggle_move_intent"
-	full_name = "Hold to toggle move intent"
-	description = "Held down to cycle to the other move intent, release to cycle back"
+	name = "examine_kb"
+	full_name = "Examine"
+	description = "Hold this key and click to examine things."
 
-/datum/keybinding/mob/toggle_move_intent/down(client/user)
+
+/datum/keybinding/mob/examine/down(client/user)
+	RegisterSignal(user.mob, COMSIG_MOB_CLICKON, .proc/examinate)
+	return TRUE
+
+
+/datum/keybinding/mob/examine/up(client/user)
+	UnregisterSignal(user.mob, COMSIG_MOB_CLICKON)
+	return TRUE
+
+
+/datum/keybinding/mob/examine/proc/examinate(datum/source, atom/A, params)
+	var/mob/user = source
+	if(!user.client || !(user.client.eye == user || user.client.eye == user.loc))
+		UnregisterSignal(user, COMSIG_MOB_CLICKON)
+		return
+	user.examinate(A)
+	var/turf/examined_turf = get_turf(A)
+	if(examined_turf && user.TurfAdjacent(examined_turf))
+		user.listed_turf = examined_turf
+		user.client.statpanel = examined_turf.name
```

Comment:
Please consider separating this into its own client preference hotkey or just being used by alt-clicking like it was before.

Reply:
Can you expand on the reasons a bit?

Reply:
From a marine standpoint the new default alt key being used to examine messes with certain inventory shortcuts like alt-clicking an MRE or pill bottle to grab the first item out from it.
Though this can be avoided by changing keybindings it ruins consistency between here and TG by having to use a different key just to pull up the turf contents menu.

The turf menu also pops up when you examine inventory items, which is a bit of a nuisance when you just want to check gun stats.
There's probably some actual cases where this could be disruptive, like trying to grab a medical scan from a patient while sorting through a pile of medkits on a nearby table.

I've also been unable to examine things using shift-click while as a ghost.

Personally just think this is a specific change that wasn't asked for, though perhaps you'll find a different opinion and support on the server.